quarobo:
  fr_trajectory_controller:
    type: effort_controllers/JointTrajectoryController
    joints:
      - FRS_joint
      - FRL_joint
      - FRF_joint
    gains:
      FRS_joint: {p: 100, d: 1, i: 1, i_clamp: 1}
      FRL_joint: {p: 100, d: 1, i: 1, i_clamp: 1}
      FRF_joint: {p: 100, d: 1, i: 1, i_clamp: 1}

  fl_trajectory_controller:
    type: effort_controllers/JointTrajectoryController
    joints:
      - FLS_joint
      - FLL_joint
      - FLF_joint
    gains:
      FLS_joint: {p: 100, d: 1, i: 1, i_clamp: 1}
      FLL_joint: {p: 100, d: 1, i: 1, i_clamp: 1}
      FLF_joint: {p: 100, d: 1, i: 1, i_clamp: 1}

  rr_trajectory_controller:
    type: effort_controllers/JointTrajectoryController
    joints:
      - RRS_joint
      - RRL_joint
      - RRF_joint
    gains:
      RRS_joint: {p: 100, d: 1, i: 1, i_clamp: 1}
      RRL_joint: {p: 100, d: 1, i: 1, i_clamp: 1}
      RRF_joint: {p: 100, d: 1, i: 1, i_clamp: 1}

  rl_trajectory_controller:
    type: effort_controllers/JointTrajectoryController
    joints:
      - RLS_joint
      - RLL_joint
      - RLF_joint
    gains:
      RLS_joint: {p: 100, d: 1, i: 1, i_clamp: 1}
      RLL_joint: {p: 100, d: 1, i: 1, i_clamp: 1}
      RLF_joint: {p: 100, d: 1, i: 1, i_clamp: 1}

    # Publish all joint state -----------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50
